content/docs/started/k8s/overview.md

```diff
@@ -13,8 +13,14 @@ existing Kubernetes cluster is from one of those, consider following those instr
 
 The Kubernetes cluster must meet the following minimum requirements:
 
-  * Kubernetes version {{% kubernetes-min-version %}} or later.
-  * At least one worker node with a minimum of:
+  * Your cluster must run Kubernetes version {{% kubernetes-min-version %}} 
```

Comment:
Would it be better to say you "your cluster must run at least version ..."
Or later seems slightly confusing because we know it won't work with 1.16 or later.

What if we just changed this to something like

"Kubeflow has been validated and tested on {{ kubernetes tested version }}""

Reply:
Thanks @jlewi. I've adjusted the wording and changed the order of the bullet points. PTAL. https://deploy-preview-1282--competent-brattain-de2d6d.netlify.com//docs/started/k8s/overview/#minimum-system-requirements